n2y/notion.py

```diff
@@ -425,7 +431,9 @@ def append_blocks(i1, i2, blocks, list):
                 children_appended.append(child_page)
                 last_i = i + 1
             elif i == len(children) - 1:
-                children_appended = append_blocks(i, len(children), children, children_appended)
```

Comment:
Is this fixing a bug? If so, please note this in the changelog and add the new version.